source/source_integration_test.go

```diff
@@ -94,7 +105,7 @@ func TestSource_Read_EmptyTable(t *testing.T) {
 	is.NoErr(err)
 }
 
-func TestSource_Snapshot_Read(t *testing.T) {
+func TestSource_snapshotRead(t *testing.T) {
```

Comment:
Is this not already covered by the acceptance tests?

Reply:
Actually, it's pretty similar.
They were written in progress. 
Do you suggest not writing simple cases here, but writing only some specific ones?

Reply:
I'd basically leave out everything which is already covered by acceptance tests.

Reply:
Done 👍